dungeon/src/starter/Starter.java

```diff
@@ -213,14 +208,19 @@ private static void taskSelectorOnSetup(Set<DSLEntryPoint> entryPoints) {
         });
   }
 
-  private static Set<DSLEntryPoint> processCLIArguments(String[] args)
-      throws IOException, ParseException {
-    Set<DSLEntryPoint> entryPoints = new HashSet<>();
-    DSLEntryPointFinder finder = new DSLEntryPointFinder();
-    DSLFileLoader.processArguments(args)
-        .forEach(path -> finder.getEntryPoints(path).ifPresent(entryPoints::addAll));
-    if (entryPoints.isEmpty()) throw new ParseException("no entry points found", 0);
-    return entryPoints;
+  private static Set<DSLEntryPoint> processCLIArguments(String[] args) throws IOException {
+    try {
+      Set<DSLEntryPoint> entryPoints = new HashSet<>();
+      DSLEntryPointFinder finder = new DSLEntryPointFinder();
+      DSLFileLoader.processArguments(args)
+          .forEach(path -> finder.getEntryPoints(path).ifPresent(entryPoints::addAll));
+      if (entryPoints.isEmpty()) {
+        throw new IOException("entry points are empty");
+      }
+      return entryPoints;
+    } catch (IOException e) {
+      throw new IOException("Couldn't open specified .dng.");
```

Comment:
theoretisch ... könnte man hier in ( ) Klammern noch den Grund der anderen IOE hinzufügen. aber wäre das für den Benutzer interessant? @cagix

Reply:
kommt drauf an, was da drin steht. probier es einfach mal aus.

Reply:
also, ich würd es gerne so lassen. dann kommt noch eine zweite oder dritte... meldung hinzu, und irgendwann wird es unübersichtlich, imho.

Reply:
warum fragst du mich dann erst, wenn du es sowieso nicht ausprobieren möchtest?

Reply:
```suggestion
      throw new IOException("Couldn't open specified .dng. " + e.getMessage());
```

Reply:
> warum fragst du mich dann erst, wenn du es sowieso nicht ausprobieren möchtest?

ich fand das nicht selbstverständlich, und es ist noch zu früh am morgen ;)

aber ich denke, so sollte es ok sein.